koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinApplication.kt

```diff
@@ -44,6 +44,14 @@ class KoinApplication private constructor() {
         return modules(listOf(modules))
     }
 
+    /**
+     * Load definitions from modules
+     * @param modules
+     */
+    fun modules(vararg modules: Module): KoinApplication {
+        return modules(modules.toList())
```

Comment:
The `modules` function's only use of the list is to call `loadModulesAndScopes` which takes an `Iterable`, so we can change `modules` to also take an `Iterable` and we'll no longer need to worry about converting to a list.

Reply:
so you say to change `module` function to take an `Iterable` ?

Reply:
Yeah. Change `fun modules(modules: List<Module>): KoinApplication {` on line 59 to `fun modules(modules: Iterable<Module>): KoinApplication {`